server/controllers/stock/required_inventory_scans.js

```diff
@@ -47,9 +47,12 @@ function getFilters(parameters) {
   filters.equals('title');
   filters.equals('description');
   filters.equals('depot_uuid');
+  filters.equals('reference_number');
```

Comment:
Question - should this be an exact match or a `LIKE` match?  I would almost prefer it if it were a `fullText()` search (using `LIKE`) since the column type is `TEXT`.  I can be persuaded otherwise though.

Reply:
Changed these to fullText filters.   See additional notes on this in the comments.